src/cmd/version.go

```diff
@@ -52,11 +52,7 @@ var versionCmd = &cobra.Command{
 		buildMap["platform"] = runtime.GOOS + "/" + runtime.GOARCH
 		buildMap["goVersion"] = runtime.Version()
 		ver, err := semver.NewVersion(config.CLIVersion)
-		if err != nil {
-			buildMap["minor"] = ""
-			buildMap["patch"] = ""
-			buildMap["prerelease"] = ""
-		} else {
+		if err == nil {
```

Comment:
```suggestion
    validSemver := err == nil
		if validSemver {
```


This is a pattern that is used in `src/internal/api/cluster.go` and I would advocate we look to switch these err == nils to reduce confusion since the more common pattern is `err != nil`

Reply:
(we could also decide on another pattern just think it should be visually distinct)